experimental/Schemes/ToricBlowups.jl

```diff
@@ -0,0 +1,67 @@
+@attributes mutable struct ToricBlowdownMorphism{
+                                                 DomainType <: NormalToricVariety, 
+                                                 CodomainType <: NormalToricVariety
+    } <: AbsSimpleBlowdownMorphism{DomainType,
+                                   CodomainType,
+                                   Nothing,
+                                   ToricBlowdownMorphism
+                                  }
```

Comment:
@HereAround : This is the new type I was talking about. Let me know what you think.

Reply:
Is good (as discussed in person).